src/components/Navbar/types/index.ts

```diff
@@ -0,0 +1,7 @@
+export type HandleMenuClickProps = (menu: string) => void
+
+export interface NavbarMenuProps {
+  handleMenuClick: HandleMenuClickProps
+  $isToggle?: boolean
+  children?: string
```

Comment:
이렇게 바꾸신다면 NavbarMenuItem에서 타입 단언을 안쓰실 수 있습니다 😊
```suggestion
  children: string
```

Reply:
> 이렇게 바꾸신다면 NavbarMenuItem에서 타입 단언을 안쓰실 수 있습니다 😊

감사합니다.. 어쩌다 옵셔널로 타입을 정의했는지는 모르겠지만 골칫덩어리였던 부분이 잘 해결되었네요 ㅎㅎ